src/Characters.tsx

```diff
@@ -1,46 +1,27 @@
-import React, { useState, useEffect } from 'react';
-import { CharacterList, Character } from './CharacterList';
-import axios from "axios"
+import { Character, CharacterList } from './CharacterList';
 import { useQuery } from 'react-query';
 
-const Characters = (): JSX.Element | null => {
+const Characters = (): JSX.Element => {
 
-  // const [jedi, setJedi] = useState<Character[]>([])
-
-
-  // const getData = async () => {
-  //   const { data } = await axios.get("https://swapi.dev/api/people/1");
-  //   setJedi([data]);
-  // }
-
-  // useEffect(() => {
-  //   if (!jedi.length) {
-  //     getData()
-  //   } else {
-  //     console.log("We have a fucking jedi!")
-  //   }
-
-  // }, [])
-
-  const { isLoading, error, data } = useQuery(["getJedi"], () => {
+  const { isLoading, data, error } = useQuery<Character>(["getJedi"], () => {
     return fetch("https://swapi.dev/api/people/1").then(res => res.json())
   });
 
-  const jedi = [data] || []
-
-  console.log({ isLoading, jedi })
-
   if (isLoading) {
     return <div>LOADING...</div>
   }
 
+  if (error) {
+    throw new Error('no character found!')
+  }
+
+  const jedi = data ? [data] : []
```

Comment:
I think this looks especially weird as we're trying to make a list out of just one character!